src/it/resources/application.conf

```diff
@@ -0,0 +1,8 @@
+process {
+  id = "test"
+}
+
+akka {
+  log-dead-letters = 0
+  loglevel = "ERROR"
```

Comment:
`eigenflow` uses errors to report retries, what produces too much noise during testing.
we could potentially improve it by setting log level only for `eigenflow` package.